### PR TITLE
Refactor Cassandra / MySQL persistence layer

### PR DESCRIPTION
**Pull request by wxing1292**
* Add dedicated handling for WorkflowMutation and WorkflowSnapshot
* Move Cassandra util code into dedicated file

### CODE REVIEW

File: common/persistence/cassandra/cassandraPersistenceUtil.go [hunk 1]
```diff
@@ -0,0 +1,1709 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```
Comment (venkat1109) [inline common/persistence/cassandra/cassandraPersistenceUtil.go]: fix copyright - should be 2019 for this new file

File: common/persistence/cassandra/cassandraPersistenceUtil.go [hunk 2]
```diff
@@ -0,0 +1,1709 @@
...
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (venkat1109) [inline common/persistence/cassandra/cassandraPersistenceUtil.go]: can you just name this file util.go ? cassandra/cassandraPersistenceUtil.go is redundant

Comment (wxing1292) [reply common/persistence/cassandra/cassandraPersistenceUtil.go]: this is only for execution table, i.e. cassandraPersistence's helper function, not visibility

Comment (wxing1292) [reply common/persistence/cassandra/cassandraPersistenceUtil.go]: btw, after this, i plan to also refactor the remaining ones, i.e. create workflow / continue as new and reset workflow execution, before 3DC reset mutable state change

**Pull request merged**